package.json

```diff
@@ -138,13 +138,6 @@
         "test:nuts": "nyc mocha \"**/*.nut.ts\" --slow 4500 --timeout 600000 --parallel",
         "version": "(oclif-dev readme && sed -i.bak -e \"s|$(echo $HOME)|~|g\" -e \"s|$(whoami)|\"\\$\\(whoami\\)\"|g\" README.md && git add README.md) || exit 0"
     },
-    "husky": {
-        "hooks": {
-            "pre-commit": "eslint \"src/**/*.ts\" \"test/**/*.ts\" --fix; if git --no-pager diff HEAD package.json | egrep '\"version\": \"[0-9]+\\.'; then npm run version; else (npm --no-git-tag-version version patch; git add package.json; sf-husky-pre-commit); fi;",
```

Comment:
probably not the `sf-husky-pre-commit` but do we want to incorporate any of this logic in the pre-commit hook?